src/components/ModalPayment/ModalAmount/ModalAmount.tsx

```diff
@@ -130,7 +168,61 @@ export const Modal = (props: Props) => {
           </ErrorMessage>
         )}
       </AmountContainer>
-
+      {amount >= 45 && (
+        <AmountContainer>
+          <Subheader>{t('paymentProcessing.amount.adopt_lantern')}</Subheader>
+          <label>{t('paymentProcessing.amount.tier_2_donation')}</label>
+          <br />
+          <br />
+          <label>{t('paymentProcessing.amount.personalize')}</label>
+          <br />
+          <br />
+          <SingleRowFormat>
+            <RowFormat width="38%">
+              <LabelText htmlFor="name">
```

Comment:
Why not create a function since the only difference is the first name/middle initial/last?

Reply:
hmm we could, i think  theres a better way of laying out the inputs